make.js

```diff
@@ -104,6 +104,7 @@ var COMMON_WEB_FILES =
       ['web/viewer.html'],
     COMMON_FIREFOX_FILES_PREPROCESS =
       [FIREFOX_CONTENT_DIR + 'PdfStreamConverter.jsm',
+       FIREFOX_CONTENT_DIR + 'PdfJsNetwork.jsm',
```

Comment:
Similar to other names "PdfJs.jsm" and "PdfJsTelemetry.jsm". We don't have consistency, do we?